client/components/code_editor/code-editor-controller.js

```diff
@@ -249,7 +249,7 @@ CodeEditorCtrl.prototype.saveTextToJson = function() {
       } catch (e) {
         // Catch errors when the JSON is invalid
         // TODO: Display error in the UI instead of in the console.
-        console.log('json error:', e.message);
+        // console.log('json error:', e.message);
```

Comment:
Are you intentionally swallowing this error? Logging seems better than nothing...